example/src/main/java/org/secondbase/example/main/HelloAll.java

```diff
@@ -21,48 +25,74 @@
     @Flag(name="counter")
     private static int counter = 1;
 
-    private static final Logger LOG = LoggerFactory.getLogger(HelloAll.class.getName());
+    private static final Logger log = LoggerFactory.getLogger(HelloAll.class.getName());
+    private static final Counter mycounter = Counter.build("mycounter", "a counter").register();
 
     private HelloAll() {}
 
-    public static void main(final String[] args) throws SecondBaseException, IOException {
+    /**
+     * Start HelloAll service.
+     */
+    public static void startHelloAllService() throws IOException {
+        mycounter.inc(counter);
+        log.info(var);
+
+        // Start a basic http server with a single endpoint.
+        final HttpServer server = HttpServer.create(new InetSocketAddress(8000), 0);
+        server.createContext("/myservice", httpExchange -> {
+            final byte[] response = "This is my service response".getBytes();
+            httpExchange.sendResponseHeaders(200, response.length);
+            final OutputStream os = httpExchange.getResponseBody();
+            os.write(response);
+            os.close();
+        });
+        server.start();
+    }
 
+    public static void main(final String[] args) throws SecondBaseException, IOException {
         final String[] realArgs = {
                 // SecondBase settings
                 "--service-name=HelloAll",
                 "--service-environment=testing",
 
-                // Consul settings
+                // Consul settings (register HelloAll service)
                 "--consul-host=localhost:8500",
                 "--service-port=8000",
-                "--consul-health-check-path=/healthz",
+                "--consul-health-check-path=/myservice",
```

Comment:
But it's not right to set it to the webconsole endpoint either, since that is a different server and the main service could be down without the webconsole being down.

I could change the HelloAll service to either be a /health endpoint, or have a /health endpoint in addition. What do you think?